frontend/components/videos/video_show_likes.jsx

```diff
@@ -14,17 +14,17 @@ class VideoShowLikes extends React.Component {
         const {numLikes, numDislikes} = this.props;
         switch (fieldVal) {
           case 1:
-            if (likeValue == 0 || likeValue == null) {
-              this.props.createLike(true, this.props.video.id);
-            } else {
+            if (likeValue == 1) {
               this.props.destroyLike(this.props.video.id);
+            } else {
```

Comment:
I would do the same as my feedback for the other switch case, deconstruct props object, and do turnery ops